api/docs/CMake_rundoxygen.cmake

```diff
@@ -256,6 +256,11 @@ if (embeddable)
   endforeach ()
```

Comment:
Describe briefly how this was tested?  I.e., that you actually did do a local jekyll check and aren't just guessing that the result looks right.

Reply:
Updated the PR description.